docs/endaq/quickstart.rst

```diff
@@ -80,3 +82,115 @@ information about the device and how it was configured.
   >>> with openFile('test.ide') as doc:
   ...     virtual_dev = endaq.device.fromRecording(doc)
 
+Quick Start Example Code
+========================
+Here is some starter code for introducing yourself to the endaq.device library. 
+Use it to familiarize yourself with with how to use endaq.device and learn 
+about some of its basic applications. Make sure to follow the 
+`installation <index.html#installation>` instructions found on the homepage 
+first. 
+
+.. code-block:: python
+   :linenos:
+   
+   """ 
+   Quick Start example code for using the endaq.device library.
+   """
+   # Import endaq.device and other useful libraries.
+   import time
+   from idelib.importer import openFile
+   import endaq.device
+
+   # Select the first device from the list of available devices
+   dev = endaq.device.getDevices()[0]
+
+   # IGNORE commandWait() for now
+
+
+   def commandWait(timeout):
```

Comment:
I think `commandWait()` can be removed... it isn't really necessary, and example code should be concise.